### ISSUE TITLE
Range Selector and Navigator unable to apply theme

### ISSUE THREAD

**Issue by briankpw** [highcharts/highcharts#11754]
#### Expected behaviour
Range Selector and Navigator should able to apply theme, `Highcharts.setOption(theme)`
 ![Correct](https://i.ibb.co/RY7kSh1/Capture.png)
https://ibb.co/7jbdzgH

#### Actual behaviour
The Range Selector’s Button Theme doesn’t apply theme, and Navigator maskFill doesn’t reflect the color. 
 ![Error](https://i.ibb.co/tHbPB3k/Capture.png)
https://ibb.co/j8LZhHB

#### Live demo with steps to reproduce

https://stackblitz.com/edit/highchart-angular-theme?file=app%2Fapp.component.ts

#### Product version

Highcharts v7.1.3 and highchart-angular 2.4.0

**Comment by KacperMadej** [comment]
Hi @briankpw 

Thank you for contacting us about the problem.

You are loading Highcharts multiple times in different components using `import` and `require`. I have tracked which instance of Highcharts is used where and I have noticed that every `require` loads a fresh instance, so theme set before are not stored. However, update in the app is causing the reuse of the previously loaded Highcharts instance with the themes set. This seems to be related to the way how the sandbox works or to the app and environment configuration, so not much could be done there to avoid this.

As a solution, you could avoid generating Highcharts with its modules with the usage of `require`. The current version of Highcharts has a feature that tracks all modules and registers them to avoid overwrites later on. This means that you do not need to worry about loading the same module multiple times as it won't be loaded if it was loaded before on the given Highcharts instance (due to the check and module registration).

tl;dr
Due to using `require` you are not referring to the same Highcharts instance across all modules in your app and themes are set on unsued for chart creation Highcharts instance.

This is not a bug in Highcharts Angular wrapper nor a bug in Highcharts code, so it will be marked so.
In case you have more questions about this please contact our technical support as this case is rather complex and won't be resolved within few comments if this comment is not enough for you to resolve the problem.
